.github/workflows/test_install_script.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       run: bundle install
 
     - name: Start Install Script for Template App
-      run: sh make.sh --bundle-id co.nimblehq.template --bundle-id-staging co.nimblehq.template.staging --project-name TemplateApp
+      run: sh make.sh --bundle-id co.nimblehq.template --bundle-id-staging co.nimblehq.template.staging --project-name TemplateApp --compile-time-warning 200
```

Comment:
I suggest creating an RFC and deciding a default `warning-compile-time` instead of input here.

Reply:
@suho it's only for test, we can choose whatever value even leave it empty to skip it

Reply:
I mean that we don't need to init a value `warning-compile-time` from the setup script, we can set it directly with `OTHER_SWIFT_FLAGS`, for example:

```swift
public func createConfiguration(projectName: String,) -> Configuration {
	let settings: [String: SettingValue] = [
		"OTHER_SWIFT_FLAGS": "-Xfrontend -warn-long-expression-type-checking=200 -Xfrontend -warn-long-function-bodies=200"
	]

	let xcconfig = Path("\(projectName)/\(path)")
	switch self {
	case .debugStaging, .debugProduction:
		return .debug(name: name, settings: settings, xcconfig: xcconfig)
	case .releaseStaging, .releaseProduction:
		return .release(name: name, settings: settings, xcconfig: xcconfig)
	}
}
```

We don't need to always use the script setup to set some value like compile code time. If we all agree with `adding compile code time` into the project, then add it directly.

Reply:
@markgravity Moreover, can we add `OTHER_SWIFT_FLAGS` into these [xcconfig files](https://github.com/nimblehq/ios-templates/tree/master/%7BPROJECT_NAME%7D/Configurations/XCConfigs) instead?